### DISCUSSION TITLE
Stay in same Column when remapping buttons

### DISCUSSION THREAD

**Discussion by N0Y0U2114** [ppy/osu#31017] [Ideas]
For inputs that can have multiple inputs (Left and right for standard for example), when it moves to the next input, it always defaults to the first input for the next row. It should instead move to the next row, but the same column as the previously mapped option.

![image](https://github.com/user-attachments/assets/110065e4-22c4-4957-8b05-b61979c685e3)

**Comment by peppy** [comment]
Makes sense IMO.
